### ISSUE TITLE
Duda respecto a BCNF en caso de herencia

### ISSUE THREAD

**Issue by FabriRandon** [IIC2413/Syllabus-2022-2#125]
Hola, tengo dudas respecto a  que si en el caso de tener herencia, (tablas que heredan de otra), la implementación del BCNF en las 'subclases tablas' sería igual que con el resto de las tablas o sería diferente?

**Comment by TamaraCucumides** [comment]
Claro, ten en cuenta que BCNF se aplica sobre las tablas, no sobre las entidades del esquema.
